scripts/run_tests.sh

```diff
@@ -19,10 +19,14 @@ set -e
 PATH=$PATH:$HOME/.local/bin
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 ROOT_DIR=$SCRIPT_DIR/..
+GO_EXECUTABLE=$(which go)
 
 cd $ROOT_DIR
 make
-go test ./...
+$GO_EXECUTABLE test ./...
+
+echo "running podman e2e-tests"
+sudo $GO_EXECUTABLE test ./pkg/server/   -v -args -podman=true
```

Comment:
Why not let the caller of the script use sudo or set the right capabilities instead?